src/components/PostComment.jsx

```diff
@@ -0,0 +1,55 @@
+import { postComments } from "../../api";
+import { useState } from 'react';
+
+const PostComment = ({article_id}) => {
+    const [loading, setLoading] = useState(false)
+    const [error, setError] = useState(null)
+    const [commentBody, setCommentBody] = useState("")
+    const [username, setUsername] = useState("")
+
+
+const handleUsernameChange = (event) => {
+    setUsername(event.target.value)
+}
+ 
+const handleCommentChange = (event) => {
+    setCommentBody(event.target.value)
+}
+
+const handlePostComment = () => {
+        setLoading(true)
+        postComments(article_id, username, commentBody)
+        .then((response) => {
+            console.log("your comment has successfully been posted")
+        })
+    .catch ((error) => {
+        setError(error.message)
+        setLoading(false)
+    })
+}
+
+return (
+    <div>
+        <h2>Leave a comment</h2>
+        <div className ="addCommentContainer">
```

Comment:
It probably makes more sense for this to be `<form>` you get the benefit of keyboard interactions working out of the box